source/extensions/filters/http/grpc_json_transcoder/config.cc

```diff
@@ -16,7 +16,7 @@ namespace GrpcJsonTranscoder {
 Server::Configuration::HttpFilterFactoryCb GrpcJsonTranscoderFilterConfig::createFilter(
     const envoy::config::filter::http::transcoder::v2::GrpcJsonTranscoder& proto_config,
     const std::string&, Server::Configuration::FactoryContext&) {
-  ASSERT(!proto_config.proto_descriptor().empty());
+  ASSERT(!(proto_config.proto_descriptor().empty() && proto_config.proto_descriptor_bin().empty()));
```

Comment:
actually can you remove these two lines ASSERT? It is already enforced by the proto validation.

Reply:
sounds good to me